### ISSUE TITLE
Firefox fails to assist NVDA when reading tables on browser mode.

### ISSUE THREAD

**Issue by joseangelyanez** [mozilla/node-firefox#46]
When having a properly defined table like in example shown below the Screen Reader (NVDA) fails to pick up the correct number of rows and cells when using the table browsing mode by pressing the "T" key on the keyboard.

**NVDA Version**
2018.1.1

**Firefox Version:**
16.0.1 (For Windows)

**Expected behavior:**
The current number of rows and cells should be read.

**Actual behavior:**
The table reads "0" rows and "0" columns.

**Remarks**
The code works on Chrome.

**Sample code**
Please see the following sample code with instructions on how to reproduce.
https://codepen.io/joseangel_yanez/pen/GBLVNo
